src/index.html

```diff
@@ -11,7 +11,216 @@
+            minlength="4"
+            maxlength="10"
+            id="password"
+            type="password"
+            autocomplete="off"
+          >
+        </div>          
+      </fieldset>
+      <fieldset class="field-set">
+        <legend>
+          An interesting fact about you!
+        </legend>
+        <div class="field">
+          Do you love cats?                
+          <input
+            type="radio"
+            name="yes"
+            id="yes"
+          >
+          <label for="yes">
+            Yes
+          </label>
+          <input
+            type="radio"
+            name="no"
+            id="no"
+          >
+          <label for="no">
+            No
+          </label>
+        </div>
+        <div class="field">
+          <label for="color">
+            What is your favorite color?
+          </label>
+          <input
+            type="color"
+            name="color"
+            id="color"
+          >
+        </div> 
+        <div class="field">
+          <label for="time">
+            What time do you go to bed?
+          </label>
+          <input
+            type="time"
+            name="timeToBed"
+            id="time"
+            autocomplete="off"
+          >
+        </div> 
+        <div class="field">
+          <label for="cars">
+            What are the favorite brand of cars?
+          </label>
+          <select         
+            name="cars"
+            id="cars"
+            required multiple
+          >
+            <option>
+              BMW
+            </option>
+            <option>
+              Audi
+            </option>
+            <option>
+              Lada
+            </option>
+          </select>
+        </div> 
+        <div>
+          <label for="range">
+            How do you rate our work?
+          </label>
+          <input
+            name="rateOfWork"
+            type="range"
+            id="range"
+            value = "0"
+          >
+        </div>           
+      </fieldset> 
+      <fieldset 
+        class="field-set"       
+        name="addInfo"
+      >
+        <legend>
+          Additional info
+        </legend> 
+        <div class="field">
+          <label for="comments">
+            Comments:
+          </label>
+          <textarea
+            name="comments"
+            id="comments"
+            cols="20"
+            rows="2"
+            autocomplete="off"
+          >
+          </textarea>
+        </div>
+        <div>
+          <label for="recommend">
+            Would you recommend us?
+          </label>
+          <select
+            name="recommends"
+            id="recommend"
+          >
+            <option>
+              yes
+            </option>
+            <option>
+              no
+            </option>
+          </select>
+        </div>        
+      </fieldset> 
+      <button type="submit">
+        Submit
+      </button>     
+    </form>    
   </body>
-</html>
+</html>
```

Comment:
```suggestion
</html>

```
Завжди потрібно додавати пустий рядок в кінці коду